src/routes/user-routes.ts

```diff
@@ -0,0 +1,28 @@
+import Elysia, { t } from "elysia";
+import { user } from "../controllers/userController";
+import { Set } from "../utils/utils";
+
+const userBodyCheck = {
+  body: t.Object({
+    id: t.Number(),
+    name: t.String(),
+    email: t.String(),
+    password: t.String(),
+  }),
+};
+
+export const userRoutes = (app: Elysia) =>
+  app.group("/users", (group) =>
+    group
+      .get("/:id", ({ params: { id }, set }) =>
+        user.getById(Number(id), set as Set)
+      )
+      .get("/", user.getAll)
+      .post("/", ({ body, set }) => user.post(body, set as Set), userBodyCheck)
```

Comment:
To create a user, you don’t need to pass an `id`. Please remove the `id` from the body

Reply:
If I try to create a user with the same id twice, I won’t get any error. And it’s bad

Reply:
1. Your name field doesn’t have any validations fields except for type. For example I can create a user with an empty name. Please add a bit more to validate name
2. Please add a sanitizer for your strings
3. Please add requirements for password, like it should be bigger than 6 symbols and less then 20 for example, to contain letter and special symbols
4. Please add a simple email validation but a bit more than just a string. Currently I can create a user with "123" email